QA.md

```diff
@@ -205,8 +205,14 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
     1. Close and open the wallet again and start a new wallet one without doing backup. It must show a yellow warning saying a backup must be done.
     1. Do the backup (following procedures in the 'Initialization' tests). The backup message has to disappear.
 
+1. **Hardware wallet migration**
+    1. Reset the wallet one more time with `localStorage.clear()`.
```

Comment:
suggestion: We could already mention the reset wallet menu from #560 instead of directing the user to the console

The only QA this step would fail is this very one we're doing right now. All future versions will have this easier path to test.